build.py

```diff
@@ -88,6 +88,13 @@ def build():
         with zipfile.ZipFile(wheel_path) as wheel:
             wheel.extract(libparsec_path, path=BASEDIR)
 
+            if build_profile == "ci" and sys.platform == "linux":
+                zipinfo = wheel.infolist()
+                for extra_lib in filter(
```

Comment:
Use a `if` in the for loopi instead of the filter, it is more pythonic and readable

```python
for item in zipinfo:
  if item.filename.startswith("parsec.libs/"):
    wheel.extract(item, path=BASEDIR)
```